### PR TITLE
Adding style to the header. This includes font and logo.

### PR DESCRIPTION
**Pull request by April-Yuen**
Here are the changes I made to the code. 
1. Added font from Google fonts.
2. Formatted the header.
3. Added a logo.

### CODE REVIEW

File: homepage/css/style.css [hunk 1]
```diff
@@ -1,12 +1,45 @@
...
+  font-size: 2.5rem;
+}
+
+.header-title{
+  color: #b31b10;
+  margin-top: 3.9%;
+}
+header {
+  display: flex;
+}
```
Comment (ivel6482) [inline homepage/css/style.css]: We could use `space-between` to push the links to the right of the screen. You may need to add a some margin on the right.

image

File: homepage/css/style.css [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
+.logo-name{
+  flex-direction: row;
+  justify-content: flex-start;
+  display: flex;
+}
+
+li{
+  list-style: none;
+  font-size: 2rem;
+}
```
Comment (ivel6482) [inline homepage/css/style.css:42]: We could add a selector for the `ul`  inside the header to have the lists sitting side by side.

image

Comment (ivel6482) [reply homepage/css/style.css:42]: We could also add some `hover` styles when the user hovers with the cursor on the link.

**Review by ivel6482 — COMMENTED** [review]
Great job! :partying_face: Left a couple of suggestions to add with some reference pictures, let me know if you have any questions.

File: homepage/css/style.css (hunk 2 above)
Comment (April-Yuen) [reply homepage/css/style.css:42]: Hi Ilba, 

I made the changes and I am going to push the code to my branch and issue another pull request. I still have no idea how to do the hamburger. I did not have time to look into that. 
Screen Shot 2022-05-15 at 4 44 42 PM

File: homepage/css/style.css [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
+  font-size: 2.5rem;
+}
+
+.header-title{
+  color: #b31b10;
+  margin-top: 3.9%;
+}
+header {
+  display: flex;
+}
```
Comment (ivel6482) [inline homepage/css/style.css]:
```suggestion
header {
  display: flex;
  justify-content: space-between;
}
```

**Review by ivel6482 — COMMENTED** [review]
Good job! Left one last little suggestion (just one line of css). After that you should be able to merge your code :tada:

**Pull request merged**